src/parsers/suite.ts

```diff
@@ -10,12 +12,22 @@ import {parseSteps} from './steps';
 import {generateExampleTableSteps, parseExampleTable} from './table';
 import {matchesTags} from './tags';
 
+interface Spec {
+    document: messages.GherkinDocument.IFeature;
+    afterEach: TestCaseHookDefinition[];
+    afterAll: TestCaseHookDefinition[];
+    beforeEach: TestCaseHookDefinition[];
+    beforeAll: TestCaseHookDefinition[];
+    skip: Boolean;
+    suites: any;
```

Comment:
I didn't dig enough to get the type of `suites`, but it's worth getting it right.